renovate.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "lockFileMaintenance": {
+      "enabled": false
```

Comment:
This is disabled by default

Reply:
By default, renovate, delete the lockfiles and install the dependencies, this sounds to me like 'if I'm going to delete the lock file every time I update, it's the same as not having a lock file.

That said, I think it makes sense to keep the lock file.

https://docs.renovatebot.com/configuration-options/#lockfilemaintenance